cfgov/core/tests/test_views.py

```diff
@@ -0,0 +1,102 @@
+import json
+import sys
+from mock import call, patch
+
+if sys.version_info[0] < 3:
+    from urlparse import urlparse
+else:
+    from urllib.parse import urlparse
+
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+
+
+class GovDeliverySubscribeTest(TestCase):
+    """
+    This TestCase mocks the following:
+    authenticated_session - removes the external govdelivery call
+    set_subscriber_topics - removes another external govdelivery call
+    assertEquals(urlparse(url).path, reverse(route_name)) was used over
+    assertRedirects because the former doesn't require the templates to exist
+    """
+
+    def test_missing_email_address(self):
+        response = self.client.post(reverse('govdelivery'),
+                                    {'code': 'FAKE_CODE'})
+        self.assertEquals(urlparse(response['Location']).path,
+                          reverse('govdelivery:user_error'))
+
+    def test_missing_gd_code(self):
+        response = self.client.post(reverse('govdelivery'),
+                                    {'email': 'fake@email.com'})
```

Comment:
Possibly update `email.com` to `example.com` here and throughout since email.com is a live spam domain, example.com is on the other hand reserved for this kind of purpose.